examples/cmd_as_argument.py

```diff
@@ -33,13 +33,15 @@ def __init__(self):
         self.multiline_commands = ['orate']
         self.maxrepeats = 3
 
-        # Add stuff to settable and shortcuts before calling base class initializer
-        self.settable['maxrepeats'] = 'max repetitions for speak command'
+        # Add stuff to shortcuts before calling base class initializer
         self.shortcuts.update({'&': 'speak'})
 
         # Set use_ipython to True to enable the "ipy" command which embeds and interactive IPython shell
         super().__init__(use_ipython=False)
 
+        # Make maxrepeats settable at runtime
+        self.settable['maxrepeats'] = 'max repetitions for speak command'
```

Comment:
Consider adding a unit test to validate that ``self.settable`` can be mutated either before or after calling ``super.__init__()``.

Reply:
This was always possible, I just updated the examples to reflect it.